### PR TITLE
Update RegisterWithPollBuddy.js

### LINKED ISSUE #311: Register form details
**Please describe what has to be done**
Need to figure out what we want on the registration. I think we want Firstname, Lastname, Email, Password, at least. Currently we have Name but backend stores Firstname, Lastname and I believe we decided we liked that better so we'll want to split it at register too.

### PR DESCRIPTION
**Pull request by mkhodak2**
### Description of change(s), including why:

Changed from just name text to first name/last name.

### Type of change(s)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Any work to still do or other concerns:

Was not running locally unsure if it correctly changed.
### Closing issues:

- Closes #311

### CODE REVIEW

File: PollBuddy-Server/frontend/src/pages/RegisterWithPollBuddy/RegisterWithPollBuddy.js [hunk 1]
```diff
@@ -76,8 +76,10 @@ class RegisterWithPollBuddy extends Component {
             To create an account, fill in the text boxes, then press submit.
           </p>
           <MDBContainer className="form-group">
-            <label htmlFor="nameText">Name:</label>
-            <input placeholder="SIS Man" className="form-control textBox" id="nameText" 
+            <label htmlFor="firstnameText">Name:</label>
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/RegisterWithPollBuddy/RegisterWithPollBuddy.js:79]: You'll want to put the labels and inputs together I believe. Label1 Input1 Label2 Input2.

File: PollBuddy-Server/frontend/src/pages/RegisterWithPollBuddy/RegisterWithPollBuddy.js [hunk 2]
```diff
@@ -76,8 +76,10 @@ class RegisterWithPollBuddy extends Component {
...
+            <label htmlFor="lastnameText">Name:</label>
+            <input placeholder="SIS" className="form-control textBox" id="firstnameText"/>
+            <input placeholder="Man" className="form-control textBox" id="lastnameText"/>
               onChange={(evt) => { this.setState({username: evt.target.value}); }}/>
```
Comment (hbh7) [inline PollBuddy-Server/frontend/src/pages/RegisterWithPollBuddy/RegisterWithPollBuddy.js:83]: This onChange part is actually part of the tag. You'll want to include it for each I believe as that's how the input validation hooks in. Duplicate the onChange part for each, and adjust the relevant fields.

**Review by hbh7 — CHANGES_REQUESTED** [review]
Just a couple things, I've left some comments with more details. Also, can you add a username field as well? Thanks

**Review by hbh7 — APPROVED** [review]
Looks good! I'll make another issue for adding validation for the name.

**Review by annieshi6 — APPROVED** [review]

**Comment by hbh7** [comment]
Frontend name validation issue: #307
Backend name validation issue: #308

**Pull request merged**